src/main/java/at/hannibal2/skyhanni/features/inventory/HarpFeatures.kt

```diff
@@ -39,12 +55,12 @@ class HarpFeatures {
         if (!openInventoryName().startsWith("Harp")) return
         val chest = event.gui as? GuiChest ?: return
 
-        for (key in keys) {
+        for ((index, key) in keys.withIndex()) {
             if (key.isKeyHeld()) {
                 if (lastClick.passedSince() > 200.milliseconds) {
                     Minecraft.getMinecraft().playerController.windowClick(
                         chest.inventorySlots.windowId,
-                        35 + key,
+                        37 + index,
```

Comment:
Why 37 and not 35?

Reply:
Because you had it with Key1, that has value 1 and Index start with 0. So in total +2

Reply:
Wait I think this should only be +1 not +2. But it somehow worked. Need to test this tomorrow

Reply:
O found why it is +2. Because Key1 starts with 2 and not 1
![image](https://github.com/hannibal002/SkyHanni/assets/85900443/c1facac3-36f1-4112-ae1d-566a360ab730)